plugins/base/src/test/kotlin/renderers/html/SearchbarDataInstallerTest.kt

```diff
@@ -0,0 +1,46 @@
+package renderers.html
+
+import org.jetbrains.dokka.base.testApi.testRunner.BaseAbstractTest
+import org.junit.Test
+import org.junit.jupiter.api.Assertions
+import utils.TestOutputWriterPlugin
+import utils.pagesJson
+
+class SearchbarDataInstallerTest: BaseAbstractTest() {
+
+    @Test
+    fun `should be displayed without leading dot`() {
```

Comment:
The detail that this bug is reproduced only with root-level declarations (those without a package) is not obvious, so I would add it to the name of the test and/or link to the bug

```suggestion
    @Test // see #2289
    fun `should display description of root declarations without a leading dot`() {
```